content/kubermatic/main/cheat-sheets/etcd/etcd-launcher/_index.en.md

```diff
@@ -1,38 +1,44 @@
 +++
-title = "Etcd Launcher"
+title = "etcd Launcher"
 weight = 30
 +++
 
-Starting with version v2.15.0, KKP introduced etcd-launcher as an experimental feature. etcd-launcher is a lightweight wrapper around the etcd binary. It's responsible for reading information from KKP API and flexibly control how the user cluster etcd ring is started.
+etcd Launcher is a lightweight wrapper around the `etcd` binary. It is responsible for reading information from KKP's Kubernetes
+API types and flexibly control how the user cluster etcd ring is started.
 
-In v2.19.0, peer TLS connections have been added to etcd-launcher. Existing etcd clusters (with or without etcd-launcher) will be upgraded to peer TLS connections if the etcd-launcher feature gate is enabled on a cluster.
+## Release Timeline
```

Comment:
Do we do something like this in our docs? Isn't this more like a release note thing?

Reply:
I've done this because this page had several references to "at this point, we added X". So this was meant to re-organise existing content. I think it's fine but you're right, it's unusual.